test/solver_tests/hybrid_solver_test.py

```diff
@@ -170,14 +196,24 @@ def test_cutoff(self):
         self.assertFalse(self.hybrid_pp_solver.assess_cutoff(["c", "d"]))
 
     def test_top_down_split_manual(self):
-        
+
         # test manually
-        mutation_frequencies = self.hybrid_pp_solver.top_solver.compute_mutation_frequencies(["a", "b", "c", "d", "e"])
+        mutation_frequencies = (
+            self.hybrid_pp_solver.top_solver.compute_mutation_frequencies(
+                [0, 1, 2, 3, 4]
+            )
+        )
```

Comment:
This is a good example of why we'd like to have `compute_mutation_frequencies` take in sample names - this is pretty unreadable in my opinion. It seems much more natural to compute the frequencies for character states across samples of interest, and here it's difficult to know what these samples are.